src/components/network-map-tab.js

```diff
@@ -712,7 +710,7 @@ export const NetworkMapTab = ({
             mapEquipments={mapEquipments}
             updatedLines={updatedLines}
             geoData={geoData}
-            waitingLoadGeoData={waitingLoadGeoData}
+            displayOverlayLoader={!basicDataReady}
```

Comment:
need to add && waiting, otherwise if there is an error the loader is displayed indefinitely

Reply:
Done